Twig/DescriptionExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ResourceBundle\Twig;
+
+use Symfony\Cmf\Component\Resource\RepositoryRegistryInterface;
+use Puli\Repository\Api\ResourceRepository;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Puli\Repository\Api\Resource\PuliResource;
+use Symfony\Cmf\Component\Resource\Description\DescriptionFactory;
+
+/**
+ * @author Daniel Leech <daniel@dantleech.com>
+ */
+class DescriptionExtension extends \Twig_Extension
+{
+    private $descriptionFactory;
+
+    /**
+     */
+    public function __construct(
+        DescriptionFactory $descriptionFactory
+    ) {
+        $this->descriptionFactory = $descriptionFactory;
+    }
+
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('cmf_resource_description', array($this, 'getDescription')),
```

Comment:
Prefer not too return false as it means reurns an object or false. With "has" it is explicit that itt returns  boolean and it allows us to9 throw a useful Exceeeption on "get". Also it does not make the logic in the example any more concise.